packages/blade/src/components/Dropdown/docs/DropdownWithSelect.stories.tsx

```diff
@@ -174,6 +174,60 @@ InternalControlledSelect.parameters = {
   },
 };
 
+// For chromatic and internal react native testing
+export const InternalControlledSingleSelect = (): JSX.Element => {
+  const [currentSelection, setCurrentSelection] = React.useState<string>('');
+
+  return (
+    <>
+      <Button
+        marginBottom="spacing.4"
+        onClick={() => {
+          setCurrentSelection('bangalore');
+        }}
+      >
+        Select Bangalore
+      </Button>
+      <Button
+        variant="secondary"
+        marginBottom="spacing.4"
+        marginLeft="spacing.4"
+        onClick={() => {
+          setCurrentSelection('');
```

Comment:
Have we documented this behavior anywhere? That you need to pass `''` to clear dropdown or reset selection. One might think to pass `null`, `undefined`, etc. for clearing the input

Reply:
I have added "Clear Selection" button in Controlled Select Input story

Reply:
https://61c19ee8d3d282003ac1d81c-splkqkkafz.chromatic.com/?path=/story/components-dropdown-with-select--with-controlled-select&globals=measureEnabled:false